indico/legacy/pdfinterface/conference.py

```diff
@@ -1236,10 +1237,20 @@ def _print_row(caption, value):
             elif item.input_type == 'multi_choice' and item.id in data:
                 multi_choice_data = ', '.join(data[item.id].friendly_data)
                 _print_row(caption=item.title, value=multi_choice_data)
+            elif item.is_section:
+                story.append(Spacer(0, 20))
+                style.fontSize = 13
+                style.textColor = (0.5, 0.5, 0.5)
+                _append_text_to_story(item.title)
+                story.append(HRFlowable(width="100%", thickness=1, color=(0.8, 0.8, 0.8), lineCap='round',
+                                        spaceAfter=10, dash=None))
+                style = ParagraphStyle({})
```

Comment:
Instead of recreating the style (and losing any settings that were set on the old one), restore the old fontSize/textColor or use a separate style (and add a new kwarg to `_append_text_to_story` to use a different style)